.github/workflows/build.yml

```diff
@@ -53,9 +53,14 @@ jobs:
 
       - name: Build with Pyodide
         run: |
-          set -vxeuo pipefail
+          set -vxeo pipefail
```

Comment:
We should probably fix the version of Python also somewhere to be the same as in Pyodide (3.11.2 at this time, I believe)